### PR TITLE
Line plot refinement

### LINKED ISSUE #15: Line plot

### PR DESCRIPTION
**Pull request by NikitaShymberg**
Added:
- Dash theme
- Checkboxes to filter the entire dataset
- Title
- Aggregating data in the line plot

Bugs:
- Altair theme still doesn't work

### CODE REVIEW

File: src/line_plot.py [hunk 1]
```diff
@@ -18,14 +18,17 @@ def generate_line_plot(data: pd.DataFrame, ycol: str):
     """
```
Comment (iamMoid) [inline src/line_plot.py:22]: You should add the 'ycol' to your parameter list in the docstring.

File: src/line_plot.py [hunk 2]
```diff
@@ -18,14 +18,17 @@ def generate_line_plot(data: pd.DataFrame, ycol: str):
...
+        label = "Average Runtime (minutes)"
+    
+    ycol = f"mean({ycol})"
 
     chart = alt.Chart(data).mark_line().encode(
-        x=alt.X("startYear", axis=alt.Axis(title="Year"), scale=alt.Scale(domain=(1970, 2020))),
-        y=alt.Y(ycol, axis=alt.Axis(title=label)),  # TODO: average this
-        color="genres"
-    )
+        x=alt.X("startYear", axis=alt.Axis(title="Year"), scale=alt.Scale(domain=(2011, 2020))),
```
Comment (iamMoid) [inline src/line_plot.py]:
```suggestion
        x=alt.X("startYear", axis=alt.Axis(title="Year", format='.0f'), scale=alt.Scale(domain=(2011, 2020))),
```

**Review by iamMoid — COMMENTED** [review]
Just a couple of minor comments

File: src/line_plot.py (hunk 1 above)
Comment (NikitaShymberg) [reply src/line_plot.py:22]: Fixed!

**Pull request merged**